earth2mip/schema.py

```diff
@@ -360,6 +364,9 @@ class EnsembleRun(pydantic.BaseModel):
     output_dir: Optional[str] = None
     output_path: Optional[str] = None
     restart_frequency: Optional[int] = None
+    grf_noise_alpha: float = 2.0
```

Comment:
Ideally these should be in some sub config class for this perturbation method, but out of the scope of this PR